backend/app.py

```diff
@@ -182,11 +227,61 @@ def csets_read(field_filter: Union[List[str], None] = Query(default=None),
             # d = [x[1:-1] for x in d]
             # return d
     else:
-        with open(CSETS_JSON_PATH, 'r') as f:
+        with open(path, 'r') as f:
             d = json.load(f)
     return d
 
 
+# todo: @Siggie: Not sure how to do what I needed in JQ, so no JQ in here yet
+@app.get("/datasets/concepts")
+def concepts_read(
```

Comment:
@Sigfried This is the route to query the `concept` table / `OMOPConcept` objects.

This gets called by the new `/concept-sets-with-concepts` route. Basically, it uses the route/function you wrote previously which gets all of the concept set data in JSON. Then, it queries this route/function w/ a list of `conceptId`, and adds those concepts to the existing JSON, and returns that to the "Cset earch" page.